src/app/utils.py

```diff
@@ -203,3 +205,53 @@ def check_license_name(name):
             return [url, exception["licenseExceptionId"]]
 
     return [False]
+
+
+def isConnected():
+    import requests
+    try:
+        response = requests.get("http://www.google.com")
+        return True
+    except requests.ConnectionError:
+        return False
+
+
+def getLicenseList(token):
+    url = "https://api.github.com/"
+    headers = {
+        "Accept":"application/vnd.github.v3.raw+json",
+        "Authorization":"bearer "+token,
+        "Content-Type":"application/json",
+    }
+    license_list_url = url+"repos/spdx/license-list-data/contents/json/licenses.json"
+    response = requests.get(license_list_url, headers=headers)
+    data = json.loads(response.text)
+    return data
+
+
+def licenseInList(namespace, namespaceId, token):
+    license_list = getLicenseList(token)
+    return_dict = {
+    "name": "",
+    "licenseId": "",
```

Comment:
There is no need to define all the keys in the `return_dict` beforehand, instead you can only add the `exists` keys to the dict and the rest will be added from lines `243 . . .` if the license matches.

Reply:
@techytushar You want me to make the URL field required?

Reply:
No, to apply a check on the URL field, so that user enters a valid URL, as it happens in the e-mail field.

Reply:
Ok, I have done that.